backend/pipeline.yaml

```diff
@@ -0,0 +1,21 @@
+serviceGroup: Microsoft.Azure.ARO.Test
+rolloutName: RP - Backend
+resourceGroups:
+- name: {{ .svc.rg }}
+  subscription: {{ .svc.subscription }}
+  aksCluster: {{ .aksName }}
+  steps:
+  - name: deploy
+    action: Shell
+    command: ["/bin/bash", "-c", "make deploy"]
```

Comment:
Why do you need to run `/bin/bash`? I think this would be OK to just have `["make", "deploy"]`

Reply:
you are right, it is fine without it. i removed it